src/controllers/v1/reports.js

```diff
@@ -23,4 +24,34 @@ module.exports = class Reports {
 			return error
 		}
 	}
+
+	/**
+	 * Get Report config
+	 * @method
+	 * @name reportFilterList
+	 * @param {Object} req - request data.
+	 * @param {String} req.query.decodedToken - user token.
+	 * @returns {JSON} - filter list.
+	 */
+
+	async reportData(req) {
+		try {
+			let entity_type = {}
+			const reportData = await reportService.getReportData(
+				req.decodedToken.id,
+				req.query.report_code,
+				req.query.report_role ? req.query.report_role : common.MENTEE_ROLE,
+				req.query.start_date ? req.query.start_date : '',
+				req.query.end_date ? req.query.end_date : '',
+				req.query.session_type ? req.query.session_type : common.ALL,
+				req.query.entity_type,
```

Comment:
@sumanvpacewisdom how ur getting entity value?

Reply:
@rakeshSgr they should send the entitytype value directly , beacause in the getfilter api they will get entityType label and value with , actually this is entities value then need to send in the entity_type